src/Loader.php

```diff
@@ -260,6 +261,39 @@ public static function add_features_section( $sections ) {
 		return $sections;
 	}
 
+	/**
+	 * Conditionally loads the beta features tracking modal.
+	 *
+	 * @param array $settings Settings.
+	 * @return array
+	 */
+	public static function maybe_load_beta_features_modal( $settings ) {
+		$tracking_enabled = get_option( 'woocommerce_allow_tracking', 'no' );
+
+		if ( 'yes' === $tracking_enabled ) {
```

Comment:
🎉 

Just making a note for the future: we may need to handle this check client-side if Settings pages get reactified since tracking can be toggled without page refresh.

Reply:
Yup, good call.